mbuild/formats/cassandramcf.py

```diff
@@ -0,0 +1,596 @@
+from __future__ import division
+
+import warnings
+
+from math import sqrt
+
+import networkx as nx
+import parmed as pmd
+
+__all__ = ['write_mcf']
+
+
+def write_mcf(structure, filename, angle_style,
+                      dihedral_style, lj14=None, coul14=None):
+    """Output a Cassandra molecular connectivity file (MCF).
+
+    Outputs a Cassandra MCF from a Parmed structure object.
+
+    Parameters
+    ----------
+    structure : parmed.Structure
+        ParmEd structure object
+    filename : str
+        Path of the output file
+    angle_style : str
+        Type of angles. 'fixed' and 'harmonic'
+        are valid choices
+    dihedral_style : str
+        Type of dihedrals. 'harmonic', 'OPLS', 'CHARMM',
+        and 'none' are valid choices
+    lj14 : float
+        Scaling factor for LJ interactions on 1-4 pairs
+    coul14 : float
+        Scaling factor for Coulombic interactions on 1-4 pairs
+
+    Notes
+    -----
+    See https://cassandra.nd.edu/index.php/documentation for
+    a complete description of the MCF format.
+
+    """
+    if not isinstance(structure, pmd.Structure):
+        raise ValueError("MCF writer requires parmed structure.")
+    if not all ([a.type for a in structure.atoms]):
+        raise ValueError("MCF writing not supported without "
+            "parameterized forcefield.")
+
+    # Conversion factors
+    IG_CONSTANT_KCAL = 0.00198720425864083 # kcal/mol*K
+    KCAL_TO_KJ = 4.184
+
+    # Check some things before we start writing the MCF
+    # Only will write MCF for Cassandra-supported options
+    if angle_style.casefold() != 'fixed' and \
```

Comment:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Your if statements use the backslash, can you switch to wrap your entire if-statement in parentheses that will span multiple lines?